AzureCalling/Views/ParticipantView.swift

```diff
@@ -15,6 +15,7 @@ class ParticipantView: UIView {
     private var renderer: VideoStreamRenderer?
     private var rendererView: RendererView?
     private var videoStreamId: String?
+    public var hasScreenShare: Bool = false
```

Comment:
I think this property should be renamed to `isScreenShare`
it should have a private setter. You could assign this property when there is videoStream Pass into. One example: 
```
func updateVideoStream(remoteVideoStream: RemoteVideoStream?) {
   self.isScreenShare = false 
}

func updateSharedScreenVideo(remoteVideoStream: RemoteVideoStream?) {
   self.isScreenShare = true 
}

....
}

Reply:
Variable renamed to isScreenSharing